_data/virendrasuryavanshi/suryavanshi.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Suryavanshi Virendrasingh # No longer than 28 characters
+institution: Indian Institute of Technology, Mandi ( IIT Mandi ) # no longer than 58 characters
+profile_pic: suryavanshi.png # Name and extension of your profile picture(ex. mona.png)
+quote: Find your passion, however long it takes, because in the end, you're the protagonist of your own life! # no longer than 110 characters
```

Comment:
Hi there, Suryavanshi!
I've looked over your PR, and it looks like there is only 1 thing that needs to be changed before it can be merged. According to README.md, your quote can include no more than 100 characters. Your quote is barely skinning the cutoff at 102 characters. If you could shorten this quote a tad, that would be great!

From README.md:
> name: FULLNAME-OR-NICKNAME # No longer than 28 characters
institution: INSTITUTION-NAME 🚩 # no longer than 58 characters
> profile_pic: mona.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
> quote: YOUR-SENIOR-QUOTE # **no longer than 100 characters**, avoid using quotes(") to guarantee the format remains the same.
> github_user: YOUR-GITHUB-USERNAME

Thanks,
-Nick

Reply:
Great catch!

Reply:
Updated.